app/assets/config/manifest.js

```diff
@@ -1,4 +1,4 @@
 //= link_tree ../images
-//= link_directory ../stylesheets .css
 //= link_tree ../../javascript .js
 //= link_tree ../../../vendor/javascript .js
+//= link_tree ../builds
```

Comment:
- [ ] Please provide a sign-up and log-in pages

- [ ] Also, you have more than one unused(empty) files in Assets folder you implemented. you can remove them to make your work clean 

![Capture](https://user-images.githubusercontent.com/63972670/196377893-6d03f718-16b6-49d6-a496-105bc38388cb.JPG)